components/Accordian/index.jsx

```diff
@@ -0,0 +1,251 @@
+        },
+        {
+          id: 7,
+          li: ' Identify and resolve performance and scalability issues',
+        },
+        {
+          id: 8,
+          li: '  Strong desire to learn and grow, with intelligence & adaptability while building the best in class systems',
+        },
+        {
+          id: 9,
+          li: 'Strong flair for effective communication and collaboration – you know when to push on and when to step back',
+        },
+        {
+          id: 10,
+          li: '  An ability to have fun while working on the hardest problems,Experience on database MySQL/ NoSQL',
+        },
+      ],
+    },
+  ];
+  return (
+    <div className="mt-6 relative w-5/6 sm:w-2/3 overflow-hidden rounded-2xl shadow-md border">
+      <input
+        type="checkbox"
+        className="peer absolute top-0 inset-x-0 w-full h-20 opacity-0 z-10 cursor-pointer"
+      />
+      <div className="h-20 w-full pl-5 pt-2 flex items-start flex-col">
+        <h1 className="text-lg font-bold text-[#7A7A7A]">{jobTitle}</h1>
+        <div className="flex space-x-2 mt-2">
+          <div className="flex">
+            <div className="">
+              <Image
+                src="/images/Location Point.png"
+                width={20}
+                height={20}
+                alt="location"
+              />
+            </div>
+            <h1 className="text-sm font-normal text-[#7A7A7A] mx-2 ">
+              {location}
+            </h1>
+          </div>
+          <div className="flex">
+            <div className="">
+              <Image
+                src="/images/Travel bag.png"
+                width={20}
+                height={20}
+                alt="location"
+              />
+            </div>
+            <h1 className="text-sm font-normal text-[#7A7A7A] mx-2 ">{exp}</h1>
+          </div>
+        </div>
+      </div>
+
+      <div className="absolute top-7 right-5 text-[#7A7A7A] transition-transform duration-500 rotate-0 peer-checked:rotate-180">
+        {/* Arrow */}
+
+        <div className="w-8 h-8 relative">
+          <Image
+            src={'/svg/arrowIcons/up-arrow.svg'}
+            alt="up-arrow"
+            layout="fill"
+            className="object-contain"
+          />
+        </div>
+      </div>
+      {/* Contents */}
+      <div className="max-h-0 peer-checked:max-h-fit  bg-white overflow-hidden transition-all duration-500">
+        <div className="space-y-5">
+          <div className="p-4">
+            <div className="space-y-3 ">
+              {titles.map(items => (
+                <div key={items.id}>
+                  <h1 className="font-bold text-[#7A7A7A]">
+                    {items.titleText}
+                    <span className="font-normal text-[#7A7A7A]">
+                      {items.jobTitle}
+                    </span>
+                  </h1>
+                </div>
+              ))}
+            </div>
+            <div>
+              {roleTitles.map((item, i) => (
```

Comment:
roleTitles, loveRoleTitles and bringAlongTitle can be put into a single file and generated through nested map